geonames/challenges/default.json

```diff
@@ -192,6 +192,7 @@
           "clients": 1
         }
       ]
+<<<<<<< HEAD
```

Comment:
this looks like an error during merge? I think you just need to remove this line, then it should be fine.